packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -74,6 +80,18 @@ describe('<AddressDetails/>', () => {
         expect(screen.queryByLabelText(address_town)).not.toBeInTheDocument();
     };
 
+    const provider_value = { is_appstore: false, is_deriv_crypto: false, is_pre_appstore: false };
+
+    const renderComponent = ({ props = mock_props, store_config = store, provider_config = provider_value }) => {
+        return render(
+            <StoreProvider store={store_config}>
+                <PlatformContext.Provider value={provider_config}>
+                    <AddressDetails {...props} />
+                </PlatformContext.Provider>
+            </StoreProvider>
+        );
+    };
+
     beforeEach(() => {
         (isDesktop as jest.Mock).mockReturnValue(true);
         (isMobile as jest.Mock).mockReturnValue(false);
```

Comment:
remove please these lines

Reply:
Here it is required because the DesktopWrapper and MobileWrapper are still making use of them